test/conftest.py

```diff
@@ -19,4 +20,15 @@ def default_text_runner(default_rules_collection):
     return RunFromText(default_rules_collection)
 
 
+@pytest.fixture
+def rule_runner(request):
+    """Return runner for a specific rule class."""
+    rule_name = request.param
+    module = import_module(f"ansiblelint.rules.{rule_name}")
```

Comment:
I don't think that imports should be delegated to the fixture.

Reply:
That is only for internal use and not doing that would make its use much more complex, especially if we start to migrate tests to reuse the runner. The hole point of this fixture was to avoid the old repeating setUp code.

Reply:
Import errors should be happening in the test modules, not in the middle of the machinery.